packages/embark/package.json

```diff
@@ -177,6 +178,7 @@
     "pump": "3.0.0",
     "remix-tests": "0.1.1",
     "request": "2.88.0",
+    "rxjs": "6.5.2",
```

Comment:
We're using `6.4.0` in other embark-* packages (`embark-whisper` and `embarkjs-whisper`). I think it would be best to have them all be the same version, i.e. bump 6.4.0 -> 6.5.2 in those other packages.